docs/services/vision/mlmodel.md

```diff
@@ -16,27 +16,31 @@ aliases:
 
 _Changed in [RDK v0.2.36 and API v0.1.118](/appendix/changelog/#vision-service)_
 
-The `mlmodel` vision service model is a machine learning detector or classifier that draws bounding boxes or returns class labels according to the specified TensorFlow Lite, TensorFlow, PyTorch, or ONNX model file available on the machine’s hard drive.
-To create a `mlmodel` detector or classifier, you need an [ML model service with a suitable model](/services/ml/).
```

Comment:
My hope in making the next section "prerequisites" and using "1." and "2." is that this will become more clear.

I think in the current form, many people will scan right over the "Before configuring your `mlmodel` detector or classifier, you need to:" and think it's just part of an explanatory intro paragraph, not realizing these are prereqs.

I think linking to the general /services/ml/ page isn't very helpful and is a little circular, whereas when we link in the prereq steps directly to /services/ml/upload-model/ that'll be a lot more helpful.

Reply:
ok